core/src/services/gcs/backend.rs

```diff
@@ -236,6 +237,13 @@ impl GcsBuilder {
         };
         self
     }
+
+    /// set the buffer size of unsized write.
```

Comment:
Please add detailed comments for this config. What's it means, the requirement (must be multiple of 256KiB), how it works..